core/domain/summary_services_test.py

```diff
@@ -540,3 +540,109 @@ def test_only_explorations_with_ratings_are_returned(self):
                            top_rated_exploration_summaries]
 
         self.assertEqual(expected_ordering, actual_ordering)
+
+
+class RecentlyPublishedExplorationDisplayableSummariesTest(
+        test_utils.GenericTestBase):
+    """Test functions for getting displayable recently published exploration
+    summary dicts.
+    """
+
+    ALBERT_NAME = 'albert'
+    ALBERT_EMAIL = 'albert@example.com'
+
+    EXP_ID_1 = 'eid1'
+    EXP_ID_2 = 'eid2'
+    EXP_ID_3 = 'eid3'
+
+    def setUp(self):
+        """Populate the database of explorations and their summaries.
+
+        The sequence of events is:
+        - (1) Albert creates EXP_ID_1.
+        - (2) Albert creates EXP_ID_2.
+        - (3) Albert creates EXP_ID_3.
+        - (4) Albert publishes EXP_ID_1.
+        - (5) Albert publishes EXP_ID_2.
+        - (6) Albert publishes EXP_ID_3.
+        - (7) Admin user is set up.
+        """
+
+        super(RecentlyPublishedExplorationDisplayableSummariesTest,
+         self).setUp()
+
+        self.admin_id = self.get_user_id_from_email(self.ADMIN_EMAIL)
+        self.albert_id = self.get_user_id_from_email(self.ALBERT_EMAIL)
+        self.signup(self.ADMIN_EMAIL, self.ADMIN_USERNAME)
+        self.signup(self.ALBERT_EMAIL, self.ALBERT_NAME)
+
+        self.save_new_valid_exploration(self.EXP_ID_1, self.albert_id)
+        self.save_new_valid_exploration(self.EXP_ID_2, self.albert_id)
+        self.save_new_valid_exploration(self.EXP_ID_3, self.albert_id)
+
+        rights_manager.publish_exploration(self.albert_id, self.EXP_ID_1)
```

Comment:
Try publishing these in a different order to when they were created, so that you can be sure that the results end up being based on the publication time and not on the initial creation time.